rules/windows/privilege_escalation_group_policy_iniscript.toml

```diff
@@ -0,0 +1,116 @@
+[metadata]
+creation_date = "2021/11/08"
+maturity = "production"
+updated_date = "2021/11/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the modification of Group Policy Objects to Add a Startup/Logon Scripts to Users or Computer Objects.
+"""
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Startup/Logon Script added to Group Policy Object"
+note = """## Triage and analysis
+
+### Investigating Scheduled Task Execution at Scale via GPO
+
+Group Policy Objects can be used by attackers as a mechanism for an attacker to instruct an arbitrarily large group of clients to
+execute specified commands at Startup, Logon, Shutdown, and Logoff. This is done by creating/modifying the `scripts.ini` or 
+`psscripts.ini` files. The scripts are stored in the following path: `<GPOPath>\\Machine\\Scripts\\`, `<GPOPath>\\User\\Scripts\\`
+
+#### Possible investigation steps:
+- This attack abuses a legitimate mechanism of the Active Directory, so it is important to determine whether the activity is legitimate
+and the administrator is authorized to perform this operation.
+- Retrieve the contents of the script file, check for any potentially malicious commands and binaries.
+- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+
+### False Positive Analysis
+- Verify if the execution is allowed and done under change management, and legitimate.
```

Comment:
I would move this to the `false_positives` meta field

Reply:
I've added an fasle_positives instance with similar content